app/Models/Role.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Role extends Model
+{
+    use HasFactory;
+}
```

Comment:
Is dit nog ergens nuttig voor?

Reply:
ja, een user heeft een eloquent relatie met Role dus die moet wel bestaan